.github/labeler.yml

```diff
@@ -0,0 +1,47 @@
+bootstrap:
+- changed-files:
+  - any-glob-to-any-file: lib/spack/spack/bootstrap/**
+ci:
+- changed-files:
+  - any-glob-to-any-file: lib/spack/spack/ci/**
+  - any-glob-to-any-file: .github/**
+  - any-glob-to-any-file: .ci/**
+commands:
+- changed-files:
+  - any-glob-to-any-file: lib/spack/spack/cmd/**
+compilers:
+- changed-files:
+  - any-glob-to-any-file: lib/spack/spack/compilers/**
```

Comment:
There's barely anything in this directory anymore, is it still worth tagging?

Reply:
@haampie or @alalazo would it be beneficial to either of your workflows to maintain a compilers tag?